src/pages/EditUserProfile.js

```diff
@@ -23,23 +25,30 @@ export const EditUserProfile = () => {
   const onFormSubmit = (event) => {
     // form not to reload page
     event.preventDefault();
+    const formData = new FormData();
+    formData.append('username', username)
+    formData.append('email', email)
+    formData.append('image', fileInput.current.files[0])
     const options = {
       method: 'PATCH',
       headers: {
```

Comment:
you deleted 'Content-Type': 'application/json', does it still need some sort of header type indication?

Reply:
it needs the authorization header and accesstoken value, but since the body of the request is a form-data and not raw JSON i removed that